go/chat/attachment.go

```diff
@@ -123,6 +138,9 @@ func putSingle(ctx context.Context, log logger.Logger, r io.Reader, size int64,
 	}
```

Comment:
The docs for ReadFull make it sound like this condition should be impossible. Should we scrap the check?